cmd/admin-replicate-info.go

```diff
@@ -75,7 +76,8 @@ func (i srInfo) String() string {
 			Field{"Endpoint", 46},
 			Field{"Sync", 4},
 			Field{"Bandwidth", 10},
-		).buildRow("Deployment ID", "Site Name", "Endpoint", "Sync", "Bandwidth"))
+			Field{"ILM Expiry Replication", 20},
+		).buildRow("Deployment ID", "Site Name", "Endpoint", "Sync", "Bandwidth", "ILM Expiry Replication"))
```

Comment:
ILM Expiry Replication - part of the header is cut off

Reply:
Will change the size of field to accomodate